src/common/options/mds.yaml.in

```diff
@@ -389,6 +389,15 @@ options:
   - mds
   flags:
   - runtime
+- name: mds_dir_prefetch
```

Comment:
Suggest that we add a yaml fragment that enables/disables this in teuthology.

See `qa/suites/fs/thrash/workloads/overrides/prefetch_dirfrags/` as an example.